src/components/Calculator.jsx

```diff
@@ -19,12 +19,17 @@ function Calculator() {
   const display = (total || '') + (operation || '') + (next || '');
 
   return (
-    <div className="calculator">
-      <div className="result">{display || '0'}</div>
-      {buttons.map((buttonValue) => (
-        <Button key={buttonValue} value={buttonValue} handleClick={handleClick} />
-      ))}
-    </div>
+    <>
+      <div className="title">
+        <h2>Let&apos;s do some math!</h2>
+      </div>
+      <div className="calculator">
+        <div className="result">{display || '0'}</div>
+        {buttons.map((buttonValue) => (
+          <Button key={buttonValue} value={buttonValue} handleClick={handleClick} />
+        ))}
+      </div>
+    </>
   );
 }
 
```

Comment:
- [ ] Your Calculator page layout does not follow the wireframe from the guidelines. Kindly fix that.

  Your Implementation | Wireframe layout
  :---:|:---:
  ![image](https://github.com/rudicarrilloypr/math-magicians/assets/36542986/0af2d556-3a91-4f3f-bcb3-927c9c7f0c7e) | ![image](https://github.com/rudicarrilloypr/math-magicians/assets/36542986/1c0d043f-f280-413c-87c6-f30deb113ec8)